### PR TITLE
cmake: enable more cmake builds

### PR DESCRIPTION
**Pull request by raiden00pl**
## Summary
cmake: enable more cmake builds
related to https://github.com/apache/nuttx/pull/9865
## Testing
CI

### CODE REVIEW

File: tools/ci/testlist/arm-05.dat [hunk 1]
```diff
@@ -26,15 +26,15 @@ CMake,nrf52840-dk:buttons
...
+CMake,nrf52840-dk:mcuboot_app
+CMake,nrf52840-dk:mcuboot_loader
 CMake,nrf52840-dk:nsh
 CMake,nrf52840-dk:ostest_tickless
 CMake,nrf52840-dk:pwm
 CMake,nrf52840-dk:qspi
-#CMake,nrf52840-dk:rndis
-# CMake,nrf52840-dk:sdc
-# CMake,nrf52840-dk:sdc_nimble
+# CMake,nrf52840-dk:rndis
```
Comment (xiaoxiang781216) [inline tools/ci/testlist/arm-05.dat:35]: can we remove all comment like nrf52840-dk:rndis.

**Review by xiaoxiang781216 — APPROVED** [review]

File: tools/ci/testlist/arm-05.dat (hunk 1 above)
Comment (raiden00pl) [reply tools/ci/testlist/arm-05.dat:35]: rndis and composite configurations need `argtable3` which is not yet ported

Comment (anchao) [reply tools/ci/testlist/arm-05.dat:35]: Hi, @raiden00pl have you doing cmake porting of argtable3, if not in your plan, I will complete argtable3 porting soon

Comment (anchao) [reply tools/ci/testlist/arm-05.dat:35]: @raiden00pl Please review PR https://github.com/apache/nuttx-apps/pull/1848

Comment (raiden00pl) [reply tools/ci/testlist/arm-05.dat:35]: @anchao great, I'll enable rndis configurations after  merging apache/nuttx-apps#1848

**Review by jerpelea — APPROVED** [review]

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**